<ui version="4.0" >
 <class>LogViewSearchWidgetBase</class>
 <widget class="QWidget" name="LogViewSearchWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>-1</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="searchLabel" >
     <property name="text" >
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchLine" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="next" >
     <property name="text" >
      <string>&amp;Next</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previous" >
     <property name="text" >
      <string>&amp;Previous</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="caseSensitive" >
     <property name="text" >
      <string>Match &amp;case</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="highlightAllButton" >
     <property name="text" >
      <string>&amp;Highlight all</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageIcon" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="message" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
